### PR TITLE
D3 filters

### PR DESCRIPTION
**Pull request by shrutiyer**

### CODE REVIEW

**Comment by davidabrahams** [comment]
Sorry, what's actually being changed in this PR? How is it different than Annabel's existing one

**Comment by aconsilvio** [comment]
It added a dropdown to select which viz you are looking it, added a function that allows similar data types to generate formatted data without rewriting tons of loops, and added a new viz

**Comment by davidabrahams** [comment]
Ah okay. I think it'll be more clear once advanced-d3 is merged

File: public/views/content/d3.html [hunk 1]
```diff
@@ -1,3 +1,15 @@
-<nvd3 options="pieOptions" data="attendanceData"></nvd3>
+<div class="vizform">
 
-<nvd3 options="histOptions" data="histData"></nvd3>
\ No newline at end of file
+<div class="form-group">
+	<label for="sel1">Data Type:</label>
+	<select class="form-control" id="sel1" ng-model="newEntry.attendance">
+	  <option value="Stars">Stars</option>
+	  <option value="Attendance">Attendance</option>
```
Comment (nmohamed) [inline public/views/content/d3.html:7]: uncapitalize values?

**Comment by nmohamed** [comment]
:dog: :dog2: :poodle:

**Pull request closed without merging**